model/client.go

```diff
@@ -584,6 +584,21 @@ func (c *Client) SearchUsers(term string, teamId string, options map[string]stri
 	}
 }
 
+// AutocompleteUsers returns two lists for autocompletion of user mentions. The first list "in",
+// specifies users in the current context (the channel, for example). The second list "out" specifies
+// users outside of that context (users not in the channel, for example). Username is required, channel
+// id is optional. Must be authenticated.
+func (c *Client) AutocompleteUsers(username string, channelId string) (*Result, *AppError) {
```

Comment:
Most of the newer ones work that way, but we can just switch this one when we get around to switching all of the old ones